apis/policies/v1alpha2/clusteradmissionpolicy_webhook.go

```diff
@@ -51,6 +51,9 @@ var _ webhook.Defaulter = &ClusterAdmissionPolicy{}
 // Default implements webhook.Defaulter so a webhook will be registered for the type
 func (r *ClusterAdmissionPolicy) Default() {
 	clusteradmissionpolicylog.Info("default", "name", r.Name)
+	if r.Spec.PolicyServer == "" {
+		r.Spec.PolicyServer = "default"
```

Comment:
I thought about making this constant... but there is only this usage still. Given that it's created in the helm chart etc, so I went ahead with hardcoding here...

Reply:
You either turn that into a constant or you disable this lint in this area of the code. I would just bite the bullet and make it a constant

Reply:
In this instance I prefer to ignore the lint. It's the same string, but they have different meanings in the different occurences...